cli/cmd/stat.go

```diff
@@ -358,14 +362,19 @@ func printSingleStatTable(stats map[string]*row, resourceType string, w *tabwrit
 	for _, key := range sortedKeys {
 		namespace, name := namespaceName(resourceType, key)
 		values := make([]interface{}, 0)
-		templateString := "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t\n"
-		templateStringEmpty := "%s\t%s\t-\t-\t-\t-\t-\t-\t\n"
+		templateString := "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t%d\t\n"
+		templateStringEmpty := "%s\t%s\t-\t-\t-\t-\t-\t-\t-\t\n"
 
-		if showTCPStats(options, resourceType) {
+		if showTCPBytes(options, resourceType) {
 			templateString = "%s\t%s\t%.2f%%\t%.1frps\t%dms\t%dms\t%dms\t%.f%%\t%d\t%.1fB/s\t%.1fB/s\t\n"
 			templateStringEmpty = "%s\t%s\t-\t-\t-\t-\t-\t-\t-\t-\t-\t\n"
 		}
 
+		if resourceType == k8s.Authority {
```

Comment:
consider `if !showTCPConns(resourceType) {` to be consistent with the code below, that way special treatment for `k8s.Authority` is isolated to one line of code.